src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -48,7 +48,7 @@ public partial class AuthenticationResult
             string idToken,
             IEnumerable<string> scopes,
             Guid correlationId,
-            AuthenticationResultMetadata authenticationResultMetadata,
+            AuthenticationResultMetadata authenticationResultMetadata = null,
```

Comment:
Without a default value, this constitutes a breaking change which we failed to detect.